frontend/src/views/CellGuide/components/CellGuideCard/index.tsx

```diff
@@ -212,12 +170,14 @@ export default function CellGuideCard({
     setCellInfoCellType(null);
   }
 
-  useEffect(() => {
-    setSelectedGene(undefined);
-  }, [selectedOrgan, selectedOrganism, setSelectedGene]);
+  const cellTypePrefix =
+    tissueName === TISSUE_AGNOSTIC ? "" : `${tissueName} specific `;
+
+  const title = `${titleize(
+    cellTypePrefix
+  )}${titleizedCellTypeName} Cell Types - CZ CELLxGENE CellGuide`;
 
-  const title = `${titleizedCellTypeName} Cell Types - CZ CELLxGENE CellGuide`;
-  const seoDescription = `Find comprehensive information about "${cellTypeName}" cell types (synonyms: ${
+  const seoDescription = `Find comprehensive information about ${cellTypePrefix}"${cellTypeName}" cell types (synonyms: ${
```

Comment:
in order to help SEO distinguish among tissue agnostic and tissue specific cell type pages, I've updated page title and seo description a little bit